sql/sql_base.cc

```diff
@@ -1914,7 +1914,12 @@ bool open_table(THD *thd, TABLE_LIST *table_list, Open_table_context *ot_ctx)
     DBUG_RETURN(FALSE);
   }
 
+#ifdef WITH_WSREP
+  if (!((flags & MYSQL_OPEN_IGNORE_FLUSH) ||
```

Comment:
I think it would be good to rewrite this using `IF_WSREP` macro.
Something along these lines:
```
IF_WSREP(!((wsrep_on(thd) && (wsrep_on(thd) && thd->wsrep_applier) ||
                     (flags & MYSQL_OPEN_IGNORE_FLUSH)),
                    !(flags & MYSQL_OPEN_IGNORE_FLUSH))
```

Reply:
I would prefer to use standard WITH_WSREP for this as condition is split between several lines, and imo it's much more readable.

Reply:
OK